CHANGELOG.md

```diff
@@ -1,12 +1,25 @@
 # Changelog
 
-## [0.3.2] - 2022-1X-XX
+## [0.3.4] - 2024-XX-XX
 
-In development
+
+## [0.3.3] - 2024-07-17
+
+**Fixes**
+  - Bug fix release for numpy 2 ({issue}`937`).
+  - Alleles are always treated as strings ({issue}`810` {pr}`928`, {user}`hyanwong`)
+
+**Breaking Changes**
+  - A permissive json schema is now set on node table metadata
+    ({issue}`416` {pr}`931`, {user}`hyanwong`).
+
+
+## [0.3.2] - 2024-07-16
 
 **Features**
 
-- `tsinfer` now supports inferring data from an `sgkit` dataset. This allows users to
+  - `tsinfer` now supports numpy2 (and 1.XX) and python 3.12.
+  - `tsinfer` now supports inferring data from an `sgkit` dataset. This allows users to
```

Comment:
Not for 0.3.2, right? I think we should remove references to sgkit here on these 0.3.x releases, and push these ideas to 0.4

Reply:
Yes, fixing this now. Yan didn't know it was a backport.